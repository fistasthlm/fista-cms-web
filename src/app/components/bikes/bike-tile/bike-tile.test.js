import React from 'react';
import { shallow } from 'enzyme';
import { Map, List } from 'immutable';
import BikeTile from './bike-tile';

const setup = propOverrides => {
    const props = Object.assign({
        bike: Map({
            images: List(),
            instagram: 'pete',
        }),
    }, propOverrides);

    return shallow(<BikeTile {...props} />);
};

describe('Render', () => {
    it('should render component', () => {
        const wrapper = setup();

        expect(wrapper).toMatchSnapshot();
    });
});
