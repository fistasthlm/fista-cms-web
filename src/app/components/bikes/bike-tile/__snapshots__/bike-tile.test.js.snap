// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render should render component 1`] = `
<div
  className="bike-tile"
>
  <Link
    replace={false}
    to="/bike/undefined"
  >
    <div
      className="bike-tile__image"
    />
    <h2
      className="bike-tile__title ellipsis"
    />
  </Link>
  <InstagramLink
    className="bike-tile__instagram-link"
    handle="pete"
  />
</div>
`;
